### ISSUE TITLE
Update Zoning for 2019 MHA

### ISSUE THREAD

**Issue by mjsisley** [openseattle/seattle-boundaries#36]
Thanks for the work on this repo.

I'm wondering if there is any way to add get the updated MHA zoning data (http://seattlecitygis.maps.arcgis.com/apps/MapSeries/index.html?appid=b0167cf4e63149e3b891307a41a639e5) added to this repo (or if there are some docs/process to follow for how this information can be collected to make a PR). I'm not seeing this data available at https://data.seattle.gov/

I'm curious where the existing zoning.geojson originated and if it is possible to get the zoning here updated for the 2019 MHA zoning changes.

**Comment by stedy** [comment]
yeah that would be a nice map to have. I will take a look into attempting to export it to geojson. I do not know where that zoning.geojson came from

**Comment by mjsisley** [comment]
Hey, thanks for the quick response. 

I also checked in with the contact listed on that map. The updated data should be posted to https://data.seattle.gov sometime this week (zoning changes officially took effect 4/19). I'm guessing the zoning.geojson file came from there. (Thanks for any attempts at getting that geojson file out.)

There is a decent amount of change of classification in the urban villages, so think would be good to have the update here once easily available.
